shell/src/main/java/alluxio/cli/DocGenerator.java

```diff
@@ -61,15 +66,22 @@ public static void main(String[] args) throws IOException {
         printHelp("Unable to parse input args: " + e.getMessage());
         return;
       }
+      if (cmd.hasOption(VALIDATE_OPTION_NAME)) {
+        validate = true;
+      }
+      if ((!cmd.hasOption(METRIC_OPTION_NAME)) && (!cmd.hasOption(CONF_OPTION_NAME))) {
```

Comment:
nit: unnecessary parenthesis surrounding the two parts?
`if (!cmd.hasOption(METRIC_OPTION_NAME) && !cmd.hasOption(CONF_OPTION_NAME)) {`